### DISCUSSION TITLE
大家的密码登录都正常吗?

### DISCUSSION THREAD

**Discussion by justid** [Mrs4s/go-cqhttp#1981] [General]
密码登录提示密码错误,不管什么协议都不行.
密码是正确的,手机qq上能登录
现在唯一能用的只有手表协议的扫码登录

**Comment by sinonliy** [comment]
使用dev版自编一下，目前安卓phone和安卓pad可以用密码登录

**Reply by justid** [reply]
直接用QQ密码登录，总是显示密码错误，已经下过ci里dev版的包了，还是一样

**Reply by justid** [reply]
config.yml密码字段用单引号包起来应该没关系吧，我去掉单引号也是一样的结果

**Comment by gwailee** [comment]
最新的安装包一样的问题，楼主消失两周了，再不出现就放弃这个了

**Reply by justid** [reply]
是账号被风控的问题, 换个qq号就行了
